architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -228,32 +234,55 @@ metadata:
   uid: ee2b9405-c68c-11e5-8a99-525400f25e34
 spec: {}
 status:
-  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample
+  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample <2>
   tags:
   - items:
     - created: 2016-01-29T13:40:11Z
       dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
       generation: 1
-      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
-    tag: latest
+      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d <3>
+    tag: latest <4>
 ----
+
+<1> The name of the image stream. 
+<2> The image source (repository or other image stream).
```

Comment:
that field can only take on a repository value, it can't be an imagestream.  i'm not really sure what purpose status.dockerImageRepository even serves, since each status.tag indicates where the tag comes from.  @soltysh ?

Reply:
It's again connected with already depreacated `.spec.dockerImageRepository`, but you're right this is just repo, you can pull image from. Usually cluster internal link, but recently with addition of public pull spec (https://github.com/openshift/origin/pull/15853) this can give you pullable docker image (from outside the cluster)